nl/tested-judge/standalone/README.md

```diff
@@ -0,0 +1,170 @@
+---
+title: TESTed zonder Dodona
+description: "TESTed zonder Dodona"
+---
+
+# TESTed zonder Dodona
+Hoewel tijdens de ontwikkeling van TESTed er gefocust werd voor het gebruik op het Dodona-platform, kan TESTed ook los
+van Dodona gebruikt worden.
+In deze handleiding zullen we uitleg hoe je TESTed kan gebruiken los van het Dodona-platform.
+
+## 1. Klonen van de TESTed GitHub repository
+De eerst stap om TESTed los van Dodona te gebruiken is het klonen of downloaden van de
+[TESTed GitHub repository](https://github.com/dodona-edu/universal-judge).
+
+![Klonen TESTed GitHub repository](./klonen_tested_repo.png)
+
+## 2. Installeren software
+De volgende stap is het installeren van de afhankelijkheden van de TESTed judge.
+
+::: tip Windows 10 tip
+Windows 10 gebruikers raden we aan om het [Windows Subsystem for Linux](https://ubuntu.com/wsl) te gebruiken.
+:::
+
+### 1) Installeren kern software
+De kern van de TESTed judge maakt gebruikt van [Python 3.9](https://docs.python.org/3/whatsnew/3.9.html).
+Hiervoor moet het `python3.9` commando beschikbaar zijn vanuit het `PATH`.
+
+TESTed heeft een aantal Python dependencies die met behulp van het
+[pip](https://en.wikipedia.org/wiki/Pip_(package_manager)) commando vanuit de root-directory van TESTed:
+```bash
+> pip3 install -r requirement.txt
+```
+
+Wanneer je de Python testen wil uitvoeren moet je ook de Python dependencies installeren voor de testen:
+```bash
+> pip3 install -r requirements-test.txt
+```
+
+### 2) Installeren specifieke programmeertaal
+:::: tabs
+::: tab Bash
+De Bash-oefening vereisen dat het [bash](https://nl.wikipedia.org/wiki/Bash) commando geïnstalleerd is en
+beschikbaar is vanuit het `PATH`.
+
+Bash is standaard geïnstalleerd op de Debian besturingssystemen en Mac OS.
```

Comment:
I wouldn't mention this; there are a lot of linux distribution, and I doubt someone will change distributions for tested.